packages/runtime/src/useSSRComputation_Server.test.tsx

```diff
@@ -1,24 +1,23 @@
-import React from 'react';
 import { renderHook } from '@testing-library/react-hooks';
 import { setErrorHandler } from "./errorHandler";
 import useSSRComputation_Server from "./useSSRComputation_Server";
-import { SSRCacheProvider } from "./SSRCacheProvider";
 
 test('useSSRComputation_Server should trigger "errorHandler" when get called with an async function', () => {
-  const asyncFn = () => Promise.resolve('result');
+  const erroneousModule = {
+    compute: () => {
+      throw new Error('Error for testing');
+    },
+  };
   const dependencies = [];
   const relativePathToCwd = '';
   const errorHandler = jest.fn();
 
   setErrorHandler(errorHandler);
-
-  const wrapper = ({ children }) => <SSRCacheProvider cache={{}}>{children}</SSRCacheProvider>;
-
-  renderHook(() => useSSRComputation_Server(asyncFn, dependencies, {}, relativePathToCwd), { wrapper });
+  renderHook(() => useSSRComputation_Server(erroneousModule, dependencies, {}, relativePathToCwd));
 
   expect(errorHandler).toHaveBeenCalledWith(
     expect.objectContaining({
-      message: expect.stringContaining('does not support async functions on the server side'),
+      message: expect.stringContaining('Error for testing'),
```

Comment:
Already removed the support for `Promises`, so this test will be only for testing the error handling. A dummy error is thrown for testing.